backends/ent/backend.go

```diff
@@ -77,7 +77,6 @@ func (backend *Backend) CloseClient() {
 
 func (backend *Backend) Debug() *Backend {
 	backend.Options.Debug = true
-	backend.client.Debug()
```

Comment:
Is this breaking something with the client? or why would we not want to debug the client?

Reply:
Yeah, at this point, client isn't initialized yet. It just sets the debug flag in the options, then in `InitClient()` this gets run:

```go
	clientOpts := []ent.Option{}
	if backend.Options.Debug {
		clientOpts = append(clientOpts, ent.Debug())
	}

	client, err := ent.Open("sqlite3", backend.Options.DatabaseFile+dsnParams, clientOpts...)
```